### DISCUSSION TITLE
9-slicing Sprites

### DISCUSSION THREAD

**Discussion by SzuWeiChen** [MyreMylar/pygame_gui#379] [Q&A] [answered]
Will 9-slicing Sprites be supported in the future?
Or is there a way to implement it now?

**Comment by MyreMylar** [comment, marked as answer]
This technique is how the rectangular shadows are implemented at the minute, but I do intend to extend it to custom images in 0.7.0 as part of a broader expansion of options for custom images on elements. I'll set up some examples when it is ready with suitably fantasy style edge windows and the like.

You could perhaps create a custom element yourself now to do some of it,  maybe utilising UIImages and anchors to get the stretching in the right places? It would be fiddly though.
